src/components/constants.js

```diff
@@ -1,5 +1,9 @@
-export const MAIN_TITLE_LABEL = 'Show Me The Weather';
+export const baseUrl = 'https://pfa.foreca.com/api/v1';
+export const myToken =
+  'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJodHRwOlwvXC9wZmEuZm9yZWNhLmNvbVwvYXV0aG9yaXplXC90b2tlbiIsImlhdCI6MTY2MzU3NjUyNywiZXhwIjo5OTk5OTk5OTk5LCJuYmYiOjE2NjM1NzY1MjcsImp0aSI6ImE5ZTliM2JkNzE4YWQ1YzAiLCJzdWIiOiJwZXR5YW5pa2xldmEiLCJmbXQiOiJYRGNPaGpDNDArQUxqbFlUdGpiT2lBPT0ifQ.zGvoW0xV9SQI6Q02eB9sUgUB-3Xnq9C_kPaLI-czLWQ';
```

Comment:
lets make sure we're making the request for the token in the app

Reply:
changes are addressed @virzhinia-petkova-redshelf